js/index.js

```diff
@@ -0,0 +1,68 @@
+$(document).ready(function(){
+
+	$('#city-entry').submit(cityEntry);
+	// when form is submitted, run function city entry
+
+	function cityEntry(event) {
+		event.preventDefault();
+		// prevent page from refreshing after entry
+		var city = $('#city-type').val();
+		// store the value of user input from city entry form as variable city
+
+		if (city === 'New York' || city === 'New York City' || city === 'NYC' || city === 'new york' || city === 'new york city' || city === 'nyc') {
+		// if variable city matches any of these new york names	
+			$('body').css('background', 'url(../FEWD22-Assignment6/images/nyc.jpg)');	
+		// change background to new york image
+			$('#city-type').val('');
```

Comment:
Since you're clearing the input value no matter what, you can run this outside of any of the conditions.